packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/fixtures/AssertionExtensions.kt

```diff
@@ -62,19 +62,20 @@ fun waitFor(timeout: Duration, interval: Duration, condition: () -> Boolean) {
 fun eventually(
     timeout: Duration = Duration.ofSeconds(10),
     recovery: () -> Unit = {},
-    fn: (Int) -> Unit
+    fn: (Int) -> Unit,
 ) {
     var attempt = 1
     waitFor(timeout, Duration.ofMillis(50)) {
         val result = runCatching {
             fn(attempt++)
-            true
-        }.getOrDefault(false)
+        }
+
+        result.exceptionOrNull()?.printStackTrace(System.err)
```

Comment:
In case there is an exception, this will print it, so we get more information about what is happening.